config/locales/en.yml

```diff
@@ -1593,17 +1606,17 @@ en:
           body: |-
             Hi %{primary_first_name},
 
-            We noticed that you have not submitted a state tax return yet with the account associated with this email address. If you submitted your state return another way, or you submitted your return under a different email address, you can disregard this message.
+            You haven't submitted your state tax return yet. As a reminder, state taxes were due by April 15. Make sure to submit your state  tax return as soon as possible in order to avoid late filing fees from %{state_name}.
 
-            This is a reminder that state taxes were due by April 15, but it's still not too late to file! Go to https://www.fileyourstatetaxes.org/en/login-options to check your return status or finish your return now. We encourage you to submit your return as soon as possible in order to avoid late filings fees from %{state_name}. Our services will be closing for the year on April 25. After this date, you will no longer be able to file through FileYourStateTaxes.
+            Go to https://www.fileyourstatetaxes.org/en/login-options to finish them now.
```

Comment:
should `https://www.fileyourstatetaxes.org/en/login-options` be a link since it's in an email?